radar-auth/src/main/java/org/radarcns/auth/authentication/AlgorithmLoader.java

```diff
@@ -0,0 +1,81 @@
+package org.radarcns.auth.authentication;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.JWTVerifier;
+import com.auth0.jwt.algorithms.Algorithm;
+import org.radarcns.auth.exception.TokenValidationException;
+import org.radarcns.auth.security.jwk.JavaWebKey;
+import org.radarcns.auth.security.jwk.JavaWebKeySet;
+import org.radarcns.auth.token.validation.ECTokenValidationAlgorithm;
+import org.radarcns.auth.token.validation.RSATokenValidationAlgorithm;
+import org.radarcns.auth.token.validation.TokenValidationAlgorithm;
+import org.radarcns.auth.token.validation.deprecated.DeprecatedEcTokenValidationAlgorithm;
+
+public class AlgorithmLoader {
```

Comment:
This could have a constructor, with the fields and methods not static. That would allow it to be loaded via Spring if needed.

Reply:
Good point. I will change it. Loading is done in constructor when the spring component starts .